offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/dataresources/persons/CustomerDbConnector.groovy

```diff
@@ -58,6 +58,7 @@ class CustomerDbConnector implements CreateCustomerDataSource, UpdateCustomerDat
   }
   
   @Override
+  @Deprecated
```

Comment:
```suggestion
  @Deprecated(since = ?, forRemoval=true)
```

Reply:
you can give the version that deprecated this part of the program. Also, it is good practice to update the JavaDoc and hint towards the method that is replacing the functionality here.

Reply:
machst du noch ne vorschlag für "since"?